jujucrashdump/crashdump.py

```diff
@@ -325,10 +324,14 @@ def _append(parent, incl):
             )
         )
 
+    @property
+    def status(self):
+        juju_status = yaml.load(open("juju_status.yaml", "r"), Loader=yaml.FullLoader)
```

Comment:
do we know that this will close the file when its done?
i remember FCE unit-tests complaining about it, but i know that isnt relevant here.